pkg/reconciler/gc/v2/gc.go

```diff
@@ -61,51 +64,67 @@ func Collect(
 		return a.After(b)
 	})
 
-	for _, rev := range revs[gcSkipOffset:] {
-		if strings.EqualFold(rev.ObjectMeta.Annotations[serving.RevisionPreservedAnnotationKey], "true") {
+	stale, nonactive := 0, 0
```

Comment:
```suggestion
	stale, inactive := 0, 0
```
I guess? :)

Reply:
inactive sounds right, but I have it this way because in my mind "non state active" is a little different than inactive which makes it sounds like something that isn't actively serving traffic. In other places we've use 'routable' as in referenced by a route (but then kind of sounds confusing because it sounds like it would mean it's accessible networking-wise) which is another candidate for naming here.

Don't get me started on the name "Configuration". The point is Knative shadows a lot of terms that already have meaning. I'm hoping to keep non-active here because that's what the config setting is called.